Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -437,13 +467,16 @@ def get_hash_classification_request(self, hash_):
         response = self._http_request(method='GET', url_suffix=endpoint_url)
         return response
 
-    def get_exclusions_request(self, item_ids=None, os_types=None, exclusion_type: str = None, limit: int = 10):
+    def get_exclusions_request(self, item_ids=None, os_types=None, exclusion_type: str = None, limit: int = 10, value_contains: str = None):
         endpoint_url = 'exclusions'
 
         params = {
             "ids": item_ids,
             "osTypes": os_types,
             "type": exclusion_type,
+            "value__contains": value_contains,
+            "includeChildren": True,
```

Comment:
The value_contains parameter here is optional and if is None there will be no filtering. Adding includeChildren and includeParent does change the behavior but only in a positive way. If those parameters are not set the API call returns a subset of the list that should be returned. I actually met with our SentinelOne rep to go over this issue and basically these are parameters that should be set to True when making this call, the integration was just configured incorrectly when set up because its not intuitive.